Src/AmrCore/AMReX_INTERP_2D.F90

```diff
@@ -331,7 +331,7 @@ subroutine AMREX_PROTECT_INTERP (fine, fine_l1,fine_l2,fine_h1,fine_h2, fblo, fb
       integer icase
 
       if (MAX(lratiox,lratioy).gt.rMAX) then
-#ifdef AMREX_DEBUG
+#if 0
```

Comment:
Do you want to avoid including `AMReX_Config.H` in this one or is this a leftover?

Reply:
We are not include any header in this file.